internal/repository/repository.go

```diff
@@ -96,7 +96,7 @@ func (r *Repository) Clone(
 			return targetDir, err
 		}
 	} else {
-		r.logger.Warn("clone already exists", slog.String("dstDir", targetDir))
+		r.logger.Info("clone already exists", slog.String("dstDir", targetDir))
```

Comment:
I an genuinely, and deeply curious as to the line of thinking that results in a cache hit being considered a warning condition 😆